pyproject.toml

```diff
@@ -1,40 +1,41 @@
 [tool.poetry]
 name = "evergreen-lint"
-version = "0.1.4"  # Duplicated in the top-level __init__.py.
+version = "0.1.5"  # Duplicated in the top-level __init__.py.
 description = ""
-authors = ["Richard Samuels <richard.samuels@mongodb.com>"]
+authors = [
+    "DevProd Build Team <devprod-build-team@mongodb.com>",
+    "DevProd Correctness Team <devprod-correctness-team@mongodb.com>",
+]
 
 [tool.poetry.scripts]
 main = "evergreen_lint.__main__:main"
 
 [tool.poetry.dependencies]
 # Python version should be set to the minimum supported toolchain version
-python = "^3.7"
+python = "^3.9"
 # These dependencies are tied to whatever versions are used by the oldest
 # supported branch of MongoDB server, presently 5.0
 PyYAML = ">=3.0.0"
-typing-extensions = "^3.10.0"
-click = "~=7.0"
+typing-extensions = ">=3.10.0"
+click = ">=7.0"
```

Comment:
I hope that changes above would make it easier to add this package as a dependency in the mongo repo.